.github/scripts/db_utils.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2022 unSkript.com
```

Comment:
Why is this file .github?? This is not github related stuff, right?

Reply:
Done. Moved it to build directory

Reply:
Will move it to toplevel directory named unskript-ctl